### PR TITLE
Use the order of enum as topic id.

### PR DESCRIPTION
**Pull request by sys1yagi**
## Issue

None

## Overview (Required)

I used the order of enum as topic id.

- At first, dd tests.
  - Add test for TopicColor. 1481b0f
  - Remove unused constant values. d114220
- then, change style.
  - Use order as topic id. fffb872
  - Remove topic id field. 1d61fe9

## Links

None

## Screenshot

None

### CODE REVIEW

File: app/src/test/java/io/github/droidkaigi/confsched2017/viewmodel/TopicColorTest.kt [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+        // invalid topic id
+        run {
+            assertThat(TopicColor.from(Topic().apply { id = -1 })).isEqualTo(TopicColor.NONE)
+            assertThat(TopicColor.from(Topic().apply { id = 10 })).isEqualTo(TopicColor.NONE)
+        }
+
+        // valid topic id
+        run {
+            assertThat(TopicColor.from(Topic().apply { id = TopicColor.PRODUCTIVITY_AND_TOOLING.ordinal }))
```
Comment (konifar) [inline app/src/test/java/io/github/droidkaigi/confsched2017/viewmodel/TopicColorTest.kt]: I think provided `id` should be concrete value like this. `id = 1`, `id = 2`, `id = 3`.

Comment (sys1yagi) [inline app/src/test/java/io/github/droidkaigi/confsched2017/viewmodel/TopicColorTest.kt]: agree.
By the way, is there documentation of topic id somewhere?
I would like to add a reference to the document to the comment.

Comment (konifar) [inline app/src/test/java/io/github/droidkaigi/confsched2017/viewmodel/TopicColorTest.kt]: Good point.
But unfortunately, we don't have any document 😢 
There is only json file... https://droidkaigi.github.io/2017/sessions.json

Comment (sys1yagi) [inline app/src/test/java/io/github/droidkaigi/confsched2017/viewmodel/TopicColorTest.kt]: oh.. 😵 
I'll add the todo comment ;)

Comment (konifar) [inline app/src/test/java/io/github/droidkaigi/confsched2017/viewmodel/TopicColorTest.kt]: Sorry... 🙇 Thanks!

Comment (sys1yagi) [inline app/src/test/java/io/github/droidkaigi/confsched2017/viewmodel/TopicColorTest.kt]: done! df02e4c ⭐️

**Comment by konifar** [comment]
Perfect! Thanks for improvement! 😄

**Pull request merged**